### DISCUSSION TITLE
Welcome to jDOSBox Discussions!

### DISCUSSION THREAD

**Discussion by Tennessene** [Tennessene/jDOSBox#15] [Announcements]
## 👋 Welcome to discussions!
  * Ask questions you’re wondering about
  * Share ideas

**Comment by illettante** [comment]
Oh  man, what I'm wondering about: that tantalising note in the readme's first paragraph about being __able to run windows apps with no OS__ ... like does that mean calling a win32 .exe from c prompt without a windows install, and having it launch the .net-based gui?! MIND EFFING BLOWN! Makes sense James's next project is BoxedWINE, as if he managed that, jDosbox would already pretty much be emulating a big chunk of windows a-la WINE.
